README.rst

```diff
@@ -42,7 +42,7 @@ To install SunPy once conda is installed run the following two commands:
 For detailed installation instructions, see the `installation guide`_ in the SunPy docs.
 
 .. _miniconda: https://docs.conda.io/en/latest/miniconda.html
-.. _installation guide: https://docs.sunpy.org/en/latest/guide/installation/index.html
+.. _installation guide: https://docs.sunpy.org/en/latest/guide/installation.html
```

Comment:
```suggestion
.. _installation guide: https://docs.sunpy.org/en/stable/guide/installation.html
```
I think this should be pointed to stable and the same for line 29.

Reply:
I've updated those links now. Yes. Should I create an issue linked to the 2.1 milestone so it is reverted then?